### PR TITLE
drift copy to clipboard - dark mode styling

### PR DESCRIPTION
**Pull request by DaedalusG**
This PR introduced a copy button for suggested fixes for each individual schema drift logOutput, it also adds styling for dark mode users

Screenshot 2023-06-14 at 2 50 53 AM
Screenshot 2023-06-14 at 2 51 05 AM

## Test plan

Introduced drift to my local db `sg start`

### CODE REVIEW

**Comment by DaedalusG** [comment]
Building on https://github.com/sourcegraph/sourcegraph/pull/50206

File: cmd/frontend/graphqlbackend/schema.graphql [hunk 1]
```diff
@@ -7101,6 +7112,19 @@ type ExternalServicesCounts {
...
+# type AutoUpgrade {
+#     """
+#     The frontend database version
+#     """
+#     version: String!
+#     """
+#     State of auto_upgrade in the frontend database
+#     """
+#     enabled: Boolean!
+# }
```
Comment (efritz) [inline cmd/frontend/graphqlbackend/schema.graphql:7127]: Is this unused?

**Review by efritz — APPROVED** [review]

File: cmd/frontend/graphqlbackend/schema.graphql (hunk 1 above)
Comment (efritz) [reply cmd/frontend/graphqlbackend/schema.graphql:7127]: Not if it's commented out lol

File: client/web/src/site-admin/SiteAdminUpdatesPage.tsx [hunk 2]
```diff
@@ -241,55 +241,83 @@ const SiteUpgradeReadiness: FunctionComponent = () => {
...
+                                                                        summary.statements.join('\n')
+                                                                    )
+                                                                } else {
+                                                                    // Handle null case
+                                                                    console.log(
+                                                                        'summary.statements was null or undefined'
+                                                                    )
+                                                                }
+                                                            }}
+                                                            variant="primary"
```
Comment (efritz) [inline client/web/src/site-admin/SiteAdminUpdatesPage.tsx:298]:
```suggestion
                                                            variant="secondary"
```

File: cmd/frontend/graphqlbackend/schema.graphql (hunk 1 above)
Comment (DaedalusG) [reply cmd/frontend/graphqlbackend/schema.graphql:7127]: :facepalm:

**Pull request closed without merging**